src/main/java/other/BlockchainDay.java

```diff
@@ -0,0 +1,32 @@
+package other;
+
+import java.util.List;
+
+public class BlockchainDay {
```

Comment:
for extra points - you can add lombok plugin and this class is going to be much shorter.

@Getter
@Setter
@AllArgsConstructor
public class BlockchainDay {